x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/AuthenticationService.java

```diff
@@ -462,11 +462,15 @@ private void consumeToken(AuthenticationToken token) {
                                 // the user was not authenticated, call this so we can audit the correct event
                                 request.realmAuthenticationFailed(authenticationToken, realm.name());
                                 if (result.getStatus() == AuthenticationResult.Status.TERMINATE) {
-                                    logger.info("Authentication of [{}] was terminated by realm [{}] - {}",
-                                        authenticationToken.principal(), realm.name(), result.getMessage());
-                                    Exception e = (result.getException() != null) ? result.getException()
-                                        : Exceptions.authenticationError(result.getMessage());
-                                    userListener.onFailure(e);
+                                    if (result.getException() != null) {
+                                        logger.info(new ParameterizedMessage("Authentication of [{}] was exceptionally terminated by " +
+                                                        "realm [{}] - {}", authenticationToken.principal(), realm.name(),
+                                                        result.getMessage()), result.getException());
+                                    } else {
+                                        logger.info("Authentication of [{}] was gracefully terminated by realm [{}] - {}",
```

Comment:
I don't think `exceptionally` and `gracefully` are accurate descriptions. In one case an exception was provided, and in another it wasn't but it doesn't follow that one is graceful and the other isn't.

Reply:
One code path amounts to calling `request.exceptionProcessingRequest` while the other calls `request.authenticationFailed`, albeit not in the same place where the `gracefully`/`exceptionally` terminology is being used.
One is more abrupt than the other.
I'm open to any suggestion wrt to terminology. For now I've removed both these keywords from the log lines.